crates/mun/test/main.mun

```diff
@@ -15,5 +14,17 @@ fn multiply(a:float, b:float):float {
 }
 
 fn main():int {
-  5
+  add(5, 3)
+}
+
+fn add_impl(a:int, b:int):int {
+    a+b
 }
+
+fn add(a:int, b:int):int {
+  add_impl(a,b)
+}
+
+fn test():int {
+  add(4,5)
+}
```

Comment:
The Git standard requires a newline at the end of a file, otherwise you'll get this warning. To solve it in IntelliJ (so probably also CLion):

https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file

Reply:
I'll chance it in my IDE but I still dont understand why its required. Its also not a default in all of my editors so why is it so important?